### DISCUSSION TITLE
Russian man

### DISCUSSION THREAD

**Discussion by CallMeVerity** [danielkrupinski/Osiris#3221] [General]
swerfaS

**Comment by PEXCH-SENTXMENT** [comment]
Да да я вас слушаю )))

**Reply by aoqia194** [reply]
I am learning Русский. I understand that this says: "Yes, yes, i'll listen to you" is this correct?

**Comment by PEXCH-SENTXMENT** [comment]
Чего-то хотели?

**Comment by PEXCH-SENTXMENT** [comment]
Может кастомную длл XDXDXD

**Comment by PEXCH-SENTXMENT** [comment]
[cheat.zip](https://github.com/danielkrupinski/Osiris/files/7086562/cheat.zip)

**Comment by PEXCH-SENTXMENT** [comment]
Там осирис просто я использую загрузчик который принимает только файлы с называнием cheat.dll

**Comment by PEXCH-SENTXMENT** [comment]
[csgo-injector.zip](https://github.com/danielkrupinski/Osiris/files/7086568/csgo-injector.zip)

**Comment by PEXCH-SENTXMENT** [comment]
Обязательно используй вак-байпас

**Comment by PEXCH-SENTXMENT** [comment]
[VAC-Bypass-Loader.zip](https://github.com/danielkrupinski/Osiris/files/7086571/VAC-Bypass-Loader.zip)

**Comment by PEXCH-SENTXMENT** [comment]
и кфг думаю сам разберешься

**Comment by PEXCH-SENTXMENT** [comment]
[Osiris.zip](https://github.com/danielkrupinski/Osiris/files/7086573/Osiris.zip)

**Comment by PEXCH-SENTXMENT** [comment]
А это мои изменненые исходники
Распакуешь и просто откроешь папку через визуал студио
https://disk.yandex.ru/d/tB3HoTzSLO3x5A
